test/module/irohad/ordering/on_demand_os_test.cpp

```diff
@@ -334,14 +334,16 @@ TEST_F(OnDemandOsTest, PassMissingTransaction) {
   auto proposal = os->onRequestProposal(target_round);
 
   // since we only sent one transaction,
-  // if the proposal is present, there is no need to check for that specific tx
+  // if the proposal is present, there is no need to check for that specific
```

Comment:
Changes in this file are not really required by clang-format and can be reverted.